config/translations/mr/about.yml

```diff
@@ -1,13 +1,13 @@
-description: 'Display basic information about Drupal Console project'
+description: 'Drupal कंसोल प्रकल्पाची प्रार्थमिक माहिती प्रदर्शित करावी'
 messages:
-    welcome: 'The Drupal Console is a suite of tools that you run on the CLI to:'
-    welcome-feature-generate: 'Generate boilerplate code.'
-    welcome-feature-interact: 'Interact with a Drupal 8 installation.'
-    welcome-feature-learn: 'Learn Drupal 8.'
-    links: 'Project links'
-    landing: 'Landing page - %s'
-    change-log: 'Change log at - %s'
-    documentation: 'Documentation at - %s'
-    support: 'Support chat room - %s'
-    supporting-organizations: 'Supporting organizations'
-    version-supported: 'Drupal supported version <info>%s</info>'
+    welcome: 'Drupal कंसोल हे एक अशा घटकांचे समूह आहे जे CLI वर चालते, त्यामुळे:'
```

Comment:
This is fixed with previous PR, Now its good to merge. folks are waiting to start with Sindhi Language.....